src/main/sensors/gyro.c

```diff
@@ -681,7 +682,7 @@ STATIC_UNIT_TESTED void performGyroCalibration(gyroSensor_t *gyroSensor, uint8_t
                 gyroSetCalibrationCycles(gyroSensor);
                 return;
             }
-            gyroSensor->gyroDev.gyroZero[axis] = (gyroSensor->calibration.sum[axis] + (gyroCalculateCalibratingCycles() / 2)) / gyroCalculateCalibratingCycles();
+            gyroSensor->gyroDev.gyroZero[axis] = gyroSensor->calibration.sum[axis] / gyroCalculateCalibratingCycles();
```

Comment:
This change looks strange. How this is related to gyro_offset?

Reply:
the gyro has a under decimal additive bias.
this ia a correction to it !

I fly LOS with HEADFREE.

At the end of a flight you have a dislocation between your yaw position 
and the Heading of your attitude quaternion.
You have to manually adjust your heading (on sight) every few minutes 
so that HEADFREE works !

Reply:
I think that answer didnt fit the question why gyroCalculateCalibratingCycles() / 2 got removed.
There need to be a reason why that code was the way it is so i was asking for why it got changed since this change doesnt seem to have a conncetion to the gyro_offset addition.

Reply:
good :) yeah. it looks useless. just wonderd if that is related to this other changes